controllers/accesslogpolicy_controller.go

```diff
@@ -96,9 +95,9 @@ func RegisterAccessLogPolicyController(
 
 	builder := ctrl.NewControllerManagedBy(mgr).
 		For(&anv1alpha1.AccessLogPolicy{}, pkg_builder.WithPredicates(predicate.GenerationChangedPredicate{})).
-		Watches(&source.Kind{Type: &gwv1beta1.Gateway{}}, handler.EnqueueRequestsFromMapFunc(r.findImpactedAccessLogPolicies), pkg_builder.WithPredicates(predicate.GenerationChangedPredicate{})).
-		Watches(&source.Kind{Type: &gwv1beta1.HTTPRoute{}}, handler.EnqueueRequestsFromMapFunc(r.findImpactedAccessLogPolicies), pkg_builder.WithPredicates(predicate.GenerationChangedPredicate{})).
-		Watches(&source.Kind{Type: &gwv1alpha2.GRPCRoute{}}, handler.EnqueueRequestsFromMapFunc(r.findImpactedAccessLogPolicies), pkg_builder.WithPredicates(predicate.GenerationChangedPredicate{}))
+		Watches(&gwv1beta1.Gateway{}, handler.EnqueueRequestsFromMapFunc(r.findImpactedAccessLogPolicies), pkg_builder.WithPredicates(predicate.GenerationChangedPredicate{})).
+		Watches(&gwv1beta1.HTTPRoute{}, handler.EnqueueRequestsFromMapFunc(r.findImpactedAccessLogPolicies), pkg_builder.WithPredicates(predicate.GenerationChangedPredicate{})).
```

Comment:
should we use gwv1 everywhere now? there are other places when we watch for beta version of gw and routes

Reply:
For max compatibility and safety I don't think we will watch v1 version resource right away. (and actually that is not required to support v1 - this PR is already able to watch them.)

Reply:
did you run any e2e tests?

Reply:
Actually I didn't and fixed the e2etest overall, e2etest will run on v1 (instead of v1beta1) resources to prove the controller is capable of handling them.